cameraview/src/main/java/com/otaliastudios/cameraview/CameraView.java

```diff
@@ -47,6 +47,7 @@
     private static final CameraLogger LOG = CameraLogger.create(TAG);
 
     public final static int PERMISSION_REQUEST_CODE = 16;
+    public final static long DEFAULT_AUTOFOCUS_RESET_DELAY_MILLIS = 3000;
```

Comment:
Can you remove the public? We don't currently expose default values for anything